server/glasses.js

```diff
@@ -0,0 +1,111 @@
+'use strict'
+
+const db = require('APP/db');
+const Glasses = db.model('glasses');
+const router = require('express').Router();
+
+
+
+//Create:
+//add glasses to database
+router.post('/admin/new-glasses', (req, res, next) => {
```

Comment:
should add a normal route like : '/' intead of '/admin/...', because we are just going to use the auth admin session.